include/TradingEngine/order_type.h

```diff
@@ -0,0 +1,22 @@
+/*
+Contains declaration for OrderType enum and Order type map.
+OrderType enum is a property of the Order class and Order
+type map is used for string representation of Order info.
+*/
+
+#ifndef TRADING_ENGINE_INCLUDE_ORDER_TYPE_H_
+#define TRADING_ENGINE_INCLUDE_ORDER_TYPE_H_
+
+
+#include <unordered_map>
+#include <string>
+
+enum class OrderType {
+	BUY,
+	SELL,
+	DEFAULT
+};
+
+extern std::unordered_map<OrderType, std::string> order_type_map;
+
+#endif // !TRADING_ENGINE_INCLUDE_ORDER_TYPE_H_
```

Comment:
btw, files usually have a new line at the end. If you noticed this red circle in the review, it means that the file doesn't have a new line at the end.

It is recommended because it makes it easier to add more logic to the file later.